plotly/tools.py

```diff
@@ -2834,26 +3663,31 @@ def _endpts_to_intervals(endpts):
     @staticmethod
     def _convert_to_RGB_255(colors):
         """
-        Return a list of tuples where each element gets multiplied by 255
+        Return a (list of) tuple(s) where each element is multiplied by 255
 
-        Takes a list of color tuples where each element is between 0 and 1
-        and returns the same list where each tuple element is normalized to be
-        between 0 and 255
+        Takes a tuple or a list of tuples where each element of each tuple is
+        between 0 and 1. Returns the same tuple(s) where each tuple element is
+        multiplied by 255
         """
-        colors_255 = []
 
-        for color in colors:
-            rgb_color = (color[0]*255.0, color[1]*255.0, color[2]*255.0)
-            colors_255.append(rgb_color)
-        return colors_255
+        if isinstance(colors, tuple):
+            return (colors[0]*255.0, colors[1]*255.0, colors[2]*255.0)
+
+        else:
+            colors_255 = []
+            for color in colors:
+                rgb_color = (color[0]*255.0, color[1]*255.0, color[2]*255.0)
+                colors_255.append(rgb_color)
+            return colors_255
```

Comment:
Same here, Re https://github.com/plotly/plotly.py/pull/471/files#r66008247 and https://github.com/plotly/plotly.py/pull/471/files#r66008288